js/src/util.js

```diff
@@ -77,6 +77,9 @@ const Util = (() => {
     },
 
     supportsTransitionEnd() {
+      if (transition === null) {
```

Comment:
you changed that again :laughing:

Reply:
Argh! This was an old branch, it seems that i've messed up this in a rebase..